campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -472,16 +473,19 @@
 
     snippet.addEventListener("show_snippet", function () {
         var btnDonate = snippet.querySelector(".button-link");
-        btnDonate.addEventListener("click", function(event) {
-            if (!btnDonate.getAttribute("disabled")) {
-                event.preventDefault();
-                event.stopPropagation();
-                btnDonate.setAttribute("disabled", "disabled");
-                addToBlockList();
-                // Give some time to IndexedDB. Bug 1236090
-                setTimeout(function() { btnDonate.click() }, 500);
-            }
-        });
+      {% if blockable and auto_blockable != false %}
```

Comment:
While this works, if somebody goes another variable in the UI for an existing snippet (e.g. the end date) and does not go to check `auto_blockable` they will unintentionally change the way the snippet acts.